shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -313,6 +362,13 @@ impl CassandraSinkCluster {
             super::connection::receive(self.read_timeout, &self.failed_requests, responses_future)
                 .await?;
 
+        let _prepare_responses = super::connection::receive(
```

Comment:
TODO: Assert that all nodes return the same id and metadata in response to PREPARE

Reply:
Assuming assert, means return or log an error instead of panicking then :+1:

Reply:
Ah yes poor choice of words given Rust's `assert` macro